CHANGELOG.md

```diff
@@ -3,6 +3,14 @@ All notable changes to this project `1.2.x` per each release will be documented
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [v1.2.0.3] - 2020-09-22
+
+### Added
+- Added support for synapse testing mode
```

Comment:
@ThilinaManamgoda shall we please add the issue link?

Also, when released please add the tested environments to the release notes. Please check the sample [change log](https://github.com/wso2/kubernetes-apim/blob/v3.2.0.2/CHANGELOG.md) and [release note](https://github.com/wso2/kubernetes-apim/releases/tag/v3.2.0.1) from here.